app/services/clients/stripe/stripe.client.js

```diff
@@ -46,7 +51,7 @@ module.exports = {
 
   retrieveAccountDetails: function (stripeAccountId) {
     return stripe.accounts.retrieve(stripeAccountId, {
```

Comment:
The interface for this method is `(id, option, config)`, this is still passing a configuration into the request options

Reply:
Stripe lib seems to have an overloaded method without account params (ie. `(id, options)`). Not sure though why it didn't work.

https://github.com/stripe/stripe-node/blob/master/types/2020-08-27/Accounts.d.ts#L3046-L3057

```
  /**
   * Retrieves the details of an account.
   */
  retrieve(
    id: string,
    params?: AccountRetrieveParams,
    options?: RequestOptions
  ): Promise<Stripe.Response<Stripe.Account>>;
  retrieve(
    id: string,
    options?: RequestOptions
  ): Promise<Stripe.Response<Stripe.Account>>;
```

Reply:
yeah you're right, briefly looking at the interface you've linked and the code looks like this should be respected